docs/smt.md

```diff
@@ -2,16 +2,16 @@
 
 - [Overview](#overview)
 - [Implementation](#implementation)
+  - [Leaf Nodes](#leaf-nodes)
   - [Inner Nodes](#inner-nodes)
   - [Extension Nodes](#extension-nodes)
-  - [Leaf Nodes](#leaf-nodes)
   - [Lazy Nodes](#lazy-nodes)
   - [Lazy Loading](#lazy-loading)
-  - [Visualisations](#visualisations)
+  - [Visualizations](#visualizations)
     - [General Trie Structure](#general-trie-structure)
     - [Lazy Nodes](#lazy-nodes-1)
 - [Paths](#paths)
-  - [Visualisation](#visualisation)
+  - [Visualization](#visualization)
```

Comment:
You really don't like british spelling ahah

Reply:
🇺🇸